release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2887,3 +2887,26 @@ link:https://access.redhat.com/solutions/6507841[{product-title} 4.8.20 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-21"]
+=== RHBA-2021:4716 - {product-title} 4.8.21 bug fix update
+
+Issued: 2021-11-23
+
+{product-title} release 4.8.21 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4716[RHBA-2021:4716] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:4715[RHBA-2021:4715] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6532121[{product-title} 4.8.21 container image list]
+
+
+[id="ocp-4-8-21-bug-fixes"]
+==== Bug fixes
+
+This fix contains changes from Kubernetes 1.21.5. For more information, see link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#v1215[Kubernetes Changelog 1.21.5].
```

Comment:
Should this be feature instead? Is this different from what we did with new Kubernetes versions in the past? 

See https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-13

Reply:
If it is the same, might be worth using the same language as in previous versions.

Reply:
@ctauchen I have incorporated the changes suggested by Steven in our release-notes channel. PTAL